content/courses/connecting-to-offchain-data/verifiable-randomness-functions.md

```diff
@@ -1341,13 +1341,18 @@ describe("burry-escrow-vrf", () => {
 
       await provider.connection.confirmTransaction(tx, "confirmed");
       console.log("Your transaction signature", tx);
-    } catch (e) {
+    } catch (error) {
       // verify tx returns expected error
       didFail = true;
-      console.log(e.error.errorMessage);
+      // Safely access error message and verify it
+      const errorMessage =
+        error?.error?.errorMessage || error.message || error.toString();
+      console.log(errorMessage);
       assert(
-        e.error.errorMessage ==
+        errorMessage.includes(
           "Current SOL price is not above Escrow unlock price.",
+        ),
+        "Unexpected error message: " + errorMessage,
```

Comment:
This is not a problem with your code, just the old code:

`error.error` doesn't make sense. If the first.`error` isn't an error message, we shouldn't be throwing it (JS technically allows us to throw Arrays or Strings or anything else, but that's considered a bad practice). Per CONTRIBUTING.md we can assume code will only throw Errors.

Just get rid of this and use: 

`if error.message.includes()` 

If that doesn't work we have bigger problems!

Reply:
fixed!